src/EditorFeatures/CSharpTest/Diagnostics/AddUsing/AddUsingTests.cs

```diff
@@ -1896,6 +1896,106 @@ static void Main(string[] args)
 @"using lowercase ; namespace A { class A { static void Main ( string [ ] args ) { var a = new b ( ) ; } } } namespace lowercase { class b { } } namespace Uppercase { class B { } } ");
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddUsing)]
```

Comment:
[WorkItem] traits, in order to remind me people about the scenarios in more detail.